### ISSUE TITLE
Rubberduck>Navigate>Find Symbol

### ISSUE THREAD

**Issue by IvenBach** [rubberduck-vba/Rubberduck#3031]
By pasting in the name of something I want to navigate to, `FooBar`, and hit enter nothing comes up. However if I delete a letter from the end, or begin typing manually it works.
```
Option Explicit

Public Sub FooBar()
    
End Sub
```

**Comment by retailcoder** [comment]
Probably something with how the autocompletion was implemented. Shouldn't be too hard to fix. Good eye!

**Comment by IvenBach** [comment]
I've done my best to describe below.

When `FindSymbolControl.xaml` is first displayed the property `SelectedItem` in the file Rubberduck.UI.FindSymbol.FindSymbolViewModel is called and is top-most on the stack. After that stack is resolved (popped?) another stack of just `SelectedItem` is created. Once both stacks have been resolved the control displays.

After the control is displayed pasting in a value to search for causes `SearchString.set()` to be at the bottom of the stack with `SelectedItem.get()` at the top. `SelectedItem` returns `null` causing any attempt to `Enter` to search for it to fail as part of `Rubberduck.UI.FindSymbol.FindSymbolControl.UIElement_OnPreviewKeyDown`'s internals calling `CanExecute()` returning `return _selectedItem != null` and thereby not searching.

If a key, like backspace, is pressed `UIElement_OnPreviewKeyDown` is at the top of a call stack. By stepping through that method another call stack with `SelectedItem.set()` at the top. This time the backing field `_selectedItem` can be set because the `value` isn't null. After that setter is finished the getter is called in its own stack. After that getter is finished, another stack with `SelectedItem.get()` at the top and `SearchString.set()` at the bottom. Stepping through this stack until `FindSymbolControl` is displayed again allows the symbol to be searched for and found since `_selectedItem` is no longer null.

It feels if https://github.com/rubberduck-vba/Rubberduck/blob/next/RetailCoder.VBE/UI/FindSymbol/FindSymbolViewModel.cs#L79 had an instruction like `SelectedItem = value;` inserted above to set the backing field this issue would be fixed. `SelectedItem` would be set when a value is pasted and before `Search` is called a couple lines below. I tried this an got the error "Cannot implicitly convert type 'string' to 'Rubberduck.UI.FindSymbol.SearchResult'".

How can a `SearchResult` be created to attempt this and see if this solves the issue?
